docs/src/index.ejs

```diff
@@ -121,6 +121,24 @@
     .hidden {
       display: none;
     }
+
+    /* Teams fonts */
+
+    @font-face {
```

Comment:
ideally would like this knowledge reside in Teams theme, so that this will be a matter of importing these fonts here from some theme's module. This will aid general scenarios where we would need to support other themes - as the approach to consume font data will be unified (to some extent, at least). What do you think?

Reply:
My proposal: leave it as is for now and avoid introducing something until we will need it. For now we have actually only one theme and I don't want to introduce something like `css-loader` to load one style.

If we have a ready simple proposal, share it 🐱